packages/freezed/lib/src/templates/copy_with.dart

```diff
@@ -177,11 +177,15 @@ $_abstractClassName${genericsParameter.append('$clonedClassName$genericsParamete
     @required String returnType,
   }) {
     String parameterToValue(Parameter p) {
-      var ternary = '${p.name} == freezed ? $accessor.${p.name} : ${p.name}';
-      if (p.type != 'Object?' && p.type != null) {
-        ternary = '$ternary as ${p.type}';
+      final ternary = '${p.name} == freezed ? $accessor.${p.name} : ${p.name}';
+      var cast = '';
+      if (p.type != null) {
+        cast = p.type.contains('?') ? '' : '!';
```

Comment:
I'd locally ignore the nnbd lint instead

The '!' operator costs an operation at runtime, which we don't want

Reply:
Alright, figured you were not very fond of ignoring lints, will do.

Reply:
Sorry I meant "ignore this line" not "ignore for file"

The lint could be useful in other places in this file, just not here